### PR TITLE
Allow Command to be initialized with values

### PR DESCRIPTION
**Pull request by hartbit**
This will allow testing SwiftPM's `BuildDelegate`.

### CODE REVIEW

File: products/llbuildSwift/BuildSystemBindings.swift [hunk 1]
```diff
@@ -199,42 +199,90 @@ extension SchedulerAlgorithm {
...
+        case values(Values)
+    }
+
+    private let representation: Representation
+
+    fileprivate init(handle: OpaquePointer?) {
+        self.representation = .handle(handle)
+    }
+
+    public init(name: String, shouldShowStatus: Bool, description: String, verboseDescription: String) {
```
Comment (ddunbar) [inline products/llbuildSwift/BuildSystemBindings.swift]: Might be nice to have a doc comment here explaining why this is useful

**Comment by aciidb0mb3r** [comment]
@hartbit the diff looks like it has the generated project?

**Comment by hartbit** [comment]
Oops, sorry. Fixed.

**Comment by hartbit** [comment]
@swift-ci please smoke test

**Comment by hartbit** [comment]
@swift-ci please smoke test

**Comment by hartbit** [comment]
@swift-ci please smoke test linux

**Comment by hartbit** [comment]
@swift-ci please smoke test

**Comment by hartbit** [comment]
Can someone merge this?

**Comment by aciidb0mb3r** [comment]
Shouldn’t you have commit access to llbuild as well?

**Comment by hartbit** [comment]
Maybe! But I don’t yet. Let me know if I should do something to get it.

**Comment by shahmishal** [comment]
@aciidb0mb3r Please email code-owners@swift.org for access.

**Pull request merged**